### PR TITLE
Update FAQ to reflect Token Launch

### PR DESCRIPTION
**Pull request by avsavsavs**
# Summary

This PR pushes a new content section from lines 559 - 627 that addresses questions related to the token launch. 

I have tried format everything as good as I possibly know/can, apologies if it needs to be reworked. 

Questions are: 
562 - Does CowSwap Have a token? 
580 - Is there a proposal to make vCOW transferrable?
585 - what is the purspose of the COW tokens?
605 - what was the criteria for the token airdrop?

### CODE REVIEW

**Comment by elena-zh** [comment]
Could I get the PR preview link please?

**Comment by avsavsavs** [comment]
https://github.com/gnosis/cowswap/pull/2540

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -573,6 +556,92 @@ export default function Faq() {
...
+          <p>
+            Traders: A trader is someone who has traded on either Ethereum mainnet (before block 13974427) or Gnosis
+            Chain (block 20024195) at least once for a total of 1K USD or who has done at least 5 trades. Note that
+            stable to stable coin trades only by factor 0.1
+          </p>
+          <p>
+            GNO Holders: A GNO holder is someone who held at least 0.1 GNO on either Ethereum mainnet (before block
+            13974427) or Gnosis Chain (block 20024195). They were eligible for holding GNO on Mainnet and/or
+            GnosisChain, and for running GBC validators. In addition, all the following LP token holders were accounted
+            for: Balancer v2 (mainnet), Uniswap v3 (mainnet), Honeyswap (gnosischain), Symetric (gnosischain), Sushiswap
```
Comment (alongoni) [inline src/custom/pages/Faq/index.tsx:639]: should we need to replace "mainnet" --> "Ethereum"? WDYT @elena-zh

Comment (anxolin) [reply src/custom/pages/Faq/index.tsx:639]: I don't think so, maybe we can discus it here https://github.com/gnosis/cowswap/pull/2556#issuecomment-1071807124

**Comment by anxolin** [comment]
Fixed the build error. We should get a link soon

**Comment by anxolin** [comment]
I fixed this small thingy too:
image

Regarding this, i find it a bit hard to read. The FAQ is already problematic with the size and i feel we are adding more. 

@michelbio @avsavsavs  should we offload some of these to the landing? If we don't want to do that, maybe we can present this in some other way that is more visual? (images, a table, smaller text, or at least some bullet points with some bold text to highlight the governance items? 

image


Same here, i think it needs some makeup before is merged:
image


Im a bit concern with how content heavy is the FAQ, what can we do to simplify it drastically?

**Comment by elena-zh** [comment]
Just some minor issues from my side: 
Space is missing here (seems that @anxolin has fixed it, but still I do not see it)
space

In some sentences periods are missing: 
period

**Comment by elena-zh** [comment]
Btw, here we are referencing to COW token address in the Mainnet. 
image

But we have the same token in GC, and then describe that campaign supported that chain. Shouldn't we show COW address somehow in the token description as well?

**Comment by anxolin** [comment]
@avsavsavs @michelbio pls let me know what to do with this. I believe we will want to include in the version of next monday, and maybe add some update regarding swpability

**Comment by anxolin** [comment]
> Space is missing here (seems that @anxolin has fixed it, but still I do not see it)

forgot to push the fix. now should be fine

**Comment by elena-zh** [comment]
@anxolin , now it is OK. Thank you!

**Pull request merged**